packages/edit-site/src/hooks/commands/use-common-commands.js

```diff
@@ -114,13 +115,64 @@ function useGlobalStylesOpenCssCommands() {
 	};
 }
 
-export function useCommonCommands() {
+function useGlobalStylesOpenRevisionsCommands() {
 	const { openGeneralSidebar, setEditorCanvasContainerView, setCanvasMode } =
 		unlock( useDispatch( editSiteStore ) );
+	const { getCanvasMode } = unlock( useSelect( editSiteStore ) );
 	const { params } = useLocation();
 	const isMobileViewport = useViewportMatch( 'medium', '<' );
-	const isListPage = getIsListPage( params, isMobileViewport );
-	const isEditorPage = ! isListPage;
+	const isEditorPage = ! getIsListPage( params, isMobileViewport );
+	const history = useHistory();
+	const { revisions } = useGlobalStylesRevisions();
```

Comment:
Since we only need to check if a site has global style revisions, what do you think about using useSelect and `getCurrentThemeGlobalStylesRevisions` directly?

The `useGlobalStylesRevisions` selects and returns much more than we actually need here.